assets/src/pages/home/page.tsx

```diff
@@ -1,14 +1,38 @@
-import { observer } from 'mobx-react-lite';
 import { Monitor } from '~/components/monitor/monitor';
 import { MonitorLoading } from '~/components/monitor/monitor-loading';
 import { useGetSupervisors } from '~/api/use-get-supervisors';
+import { useEffect, useState } from 'react';
 
-export const MainPage = observer(() => {
-  const { data: supervisors, isLoading } = useGetSupervisors();
+let serverTimeDiffIntervalId = 0;
+let prevDataEncoded = '';
+
+export const MainPage = () => {
+  const [serverTimeDiff, setServerTimeDiff] = useState(0);
+  clearInterval(serverTimeDiffIntervalId);
+  serverTimeDiffIntervalId = setInterval(() => setServerTimeDiff(serverTimeDiff + 1), 1000);
```

Comment:
do it in useEffect please

Reply:
I tried, its not working, because 2 runs, first run will remove interval from 1 run and set self, but it not used. So timer not incrementing